js/index.js

```diff
@@ -1,6 +1,91 @@
+} else if (hacker1.localeCompare(hacker2) > 0) {
+  console.log(`Yo, the navigator goes first definitely.`);
+} else {
+  console.log(`What?! You both have the same name?`);
+}
+
+// Bonus 1:
+
+let lorumIpsum = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi pharetra, turpis id luctus tempus, orci lacus mollis sapien, eget imperdiet dui metus sodales nibh. Donec dolor tortor, vehicula et augue vitae, efficitur condimentum nisl. Vestibulum id pellentesque ligula. Nunc sed pharetra purus. Nam gravida dolor eu ipsum dapibus placerat. Etiam tellus leo, ultricies non consequat eu, ullamcorper a orci. Pellentesque cursus luctus diam eget porttitor. Vivamus consectetur vehicula mauris quis fringilla. Ut lobortis orci a tincidunt feugiat. Vestibulum interdum tempus eros, vel ultrices elit rhoncus id. Phasellus faucibus mauris nec nisi vestibulum laoreet. Curabitur ipsum erat, maximus vitae luctus et, hendrerit ut nunc. Nullam vitae ante orci. Quisque malesuada purus leo, a rutrum justo pellentesque sed. Sed eget consectetur magna. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Maecenas imperdiet vel lorem a consectetur. Maecenas at risus ultrices, accumsan ipsum sit amet, cursus mauris. Praesent gravida libero justo, id dignissim metus gravida et. Nunc nec elit leo. Mauris non vulputate turpis. Donec ultrices neque eu lectus bibendum elementum. Nulla facilisi. Morbi quis ligula vel eros tristique vulputate non vitae nunc. Sed eget orci arcu. Pellentesque id nisi quam. Pellentesque vitae tellus eu leo porttitor varius. Sed semper venenatis sem vel cursus. Suspendisse eu sapien sollicitudin, elementum orci nec, pretium felis. Sed pellentesque, eros in consectetur rutrum, massa velit placerat metus, et fermentum metus eros non dolor. Proin venenatis enim et libero imperdiet blandit. Vestibulum laoreet ex pellentesque quam pellentesque, et eleifend elit commodo. Aliquam at efficitur tortor, pulvinar consectetur justo. Aliquam tempor massa eget lectus tempor maximus. Sed in quam ac neque tempor varius. Quisque sem dui, condimentum in dui malesuada, cursus vulputate massa.";
+
+// Counting words:
+
+wordCount = lorumIpsum.split(" ").length;
+console.log(wordCount);
+
+// Counting 'et':
+
+function countEt (lorumIpsum) {
+  let count = 0;
+  for (let i = 0; i < lorumIpsum.length; i++)
+  if((lorumIpsum[i] == " ") && (lorumIpsum[i+1] == "e") && (lorumIpsum[i+2] == "t") && (lorumIpsum[i+3] == " " || "." || ",")) {
+      count += 1;
+    }
+  console.log(count);
+}
+
+countEt(lorumIpsum)
+
+// Bonus 2:
+
+function checkPalindrome(phraseToCheck){
+    const phraseLength = phraseToCheck.length;
+      
+      for (let i = 0; i < phraseLength / 2; i++) {
+        if (phraseToCheck[i] !== phraseToCheck[phraseLength - 1 - i]) {
+          console.log('It is not a palindrome')
+        } else {
+            console.log('It is a palindrome')
+        }
+      }
+  }  
+  
+  checkPalindrome("value")
```

Comment:
Here it is best to look again at the solutions we will send you, but the rest is excellent

Reply:
Yeah I would love to see the solutions, because I got really stuck at the end. I've tried to work with break and continue, but then I got errors. :(